music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -109,7 +109,14 @@ async def get_config_entries(
     values: the (intermediate) raw values for config entries sent with the action.
     """
     returncode, output = await check_output(["snapserver", "-v"])
-    snapserver_present = returncode == 0 and "snapserver v0.27.0" in output.decode()
+    snapserver_version: int = int(output.decode().split(".")[1] or -1)
```

Comment:
this will crash if the returncode is not 0 so you will need to move this check after the returncode check or guard for empty output

Reply:
Something like this is ok?
` snapserver_version: int = int(output.decode().split(".")[1]) if returncode == 0 else -1`

Reply:
yes